### ISSUE TITLE
sub-category chooser does not have hover states or keyboard nav

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#8698]
## Observed behavior

The sub-category chooser has no keyboard navigation or hover states:

![2021-11-15 15 21 04](https://user-images.githubusercontent.com/2367265/141848943-6c6a19ca-abc6-4a0d-8cd6-b893d66e1371.gif)

In some cases there is a pointer cursor, but the item isn't actually clickable:

![2021-11-15 15 28 48](https://user-images.githubusercontent.com/2367265/141849409-bba5fb37-8f11-4fc8-b693-f1af9b2d0532.gif)

## Expected behavior

The UI in here feels fairly different from typical interactive elements in KDS. If possible we should try to use established components, but if the design calls for something special we'll need do a bit more work on ensuring accessibility.

## User-facing consequences

* no keyboard nav
* not totally clear that headers _and_ child elements are clickable categories
* some things that look clickable due to cursor aren't actually

## Context

0.15.0-beta2
